Layout.tsx

```diff
@@ -0,0 +1,17 @@
+/** @jsx h */
+import { h } from "preact";
+import { PropsWithChildren } from "https://esm.sh/v94/preact@10.10.0/compat/src/index.d.ts";
```

Comment:
```suggestion
import { PropsWithChildren } from "preact/compat";
```

Does this work?

If not, move this url to `import_map` and use another symbol here?